### PR TITLE
Add Safe addresses and Blockscout endpoints for Ethereum Classic

### PR DESCRIPTION
**Pull request by ziogaschr**

### CODE REVIEW

**Comment by Uxio0** [comment]
Thanks! You need to accept the CLA

**Comment by ziogaschr** [comment]
I have read the CLA Document and I hereby sign the CLA

File: gnosis/safe/addresses.py [hunk 1]
```diff
@@ -317,6 +317,12 @@
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 2987582, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 2987584, "1.3.0"),
     ],
+    EthereumNetwork.ETHEREUM_CLASSIC_MAINNET: [
+        ("0x69f4D1788e39c87893C980c06EdF4b7f686e2938", 15904946, "1.3.0"),
```
Comment (Uxio0) [inline gnosis/safe/addresses.py:321]: Please add L2 version too

Comment (ziogaschr) [reply gnosis/safe/addresses.py:322]: Done with [7d6fff7](https://github.com/safe-global/safe-eth-py/pull/540/commits/7d6fff760911f7046391372f6beed50744ae6b2f), thanks

**Review by Uxio0 — APPROVED** [review]

**Pull request merged**